api/tests/test_account_views.py

```diff
@@ -31,18 +37,24 @@ def test_account_detail_no_account(api_client):
     assert response.status_code == 404
 
 
-def test_account_put_invalid(ratom_file, api_client):
+def test_account_put_invalid(ratom_file, api_client, file_serializer_validation_mock):
     # Test PUT with invalid serializer
     url = reverse("account_detail", args=[ratom_file.account.pk])
     data = {"filename": "x" * 201}
+    instance = file_serializer_validation_mock.return_value
+    instance.exists = False
     response = api_client.put(url, data=data)
     assert response.status_code == 400
```

Comment:
Ideally we'd check the response here too:

```python
    assert (
        str(response.data["filename"][0])
        == "Ensure this field has no more than 200 characters."
    )
```

Or perhaps moving these more discrete tests into a serializer-based test.

Reply:
Same as above the False should be True which really just keeps the azure call from mucking things up. But you are also correct here again I should test that the response has the character length error.